src/UCommerce.Transactions.Payments.Braintree/BraintreePaymentMethodService.cs

```diff
@@ -90,14 +102,15 @@ public override Payment RequestPayment(PaymentRequest paymentRequest)
         /// <param name="payment">The payment to process.</param>
         public override void ProcessCallback(Payment payment)
         {
+            if (payment.PaymentStatus.PaymentStatusId != (int)PaymentStatusCode.PendingAuthorization)
+                return;
+

```

Comment:
I'd prefer the variant with brackets around, but if you insist, you can leave it that way.

Reply:
It's a style that was chosen and used in other providers. I decided to keep it the same, for consistency.